src/commands/dev/index.js

```diff
@@ -342,14 +342,13 @@ async function startDevServer(settings, log) {
 
   log(`${NETLIFYDEVLOG} Starting Netlify Dev with ${settings.type}`)
   const args = settings.command === 'npm' ? ['run', ...settings.args] : settings.args
-  const ps = child_process.spawn(await which(settings.command), args, {
+  const commandBin = await which(settings.command)
+  const ps = child_process.spawn(commandBin, args, {
     env: { ...settings.env, FORCE_COLOR: 'true' },
-    stdio: settings.stdio || 'inherit',
-    detached: true,
-    shell: true,
+    stdio: 'pipe',
```

Comment:
Is it possible for `settings.command` to ask for interactive input from the user?
If so, `stdio: 'pipe'` would not work.

Reply:
How about we do `process.stdin.pipe(ps.stdin)`?

Reply:
Implemented here: https://github.com/netlify/cli/pull/823/files